test_image_analysis.py

```
@@ -0,0 +1,250 @@
+    url = "http://localhost:8080/api/chat"
+
+    payload = {
+        "ask": "What is 2+2?",
+        "model": "gpt-4o-mini",
+        "conversation_history": [
+            {"role": "system", "content": "You are a helpful assistant."}
+        ]
+    }
+
+    try:
+        response = requests.post(url, json=payload)
```

Ruff (0.14.10)

Probable use of `requests` call without timeout

(S113)